src/test/scala/mesosphere/marathon/api/akkahttp/v2/GroupsControllerTest.scala

```diff
@@ -125,35 +116,30 @@ class GroupsControllerTest extends UnitTest with ScalatestRouteTest with Inside
       }
     }
 
-    "show app in a given version for root group" in {
-      val infoService = mock[GroupInfoService]
-      infoService.selectGroupVersion(eq(PathId.empty), any, any, any) returns Future.successful(Some(GroupInfo(createGroup(PathId.empty), None, None, None)))
+    "show app in a given version for root group" in new Fixture {
+      infoService.selectGroupVersion(org.mockito.Matchers.eq(PathId.empty), any, any, any) returns Future.successful(Some(GroupInfo(createGroup(PathId.empty), None, None, None)))
       val f = new Fixture(infoService = infoService)
 
-      Get(Uri./.withPath(Path("/versions/2017-10-30T16:08:53.852Z"))) ~> f.groupsController.route ~> check {
```

Comment:
Hm, if `f` is removed `val f = new Fixture(infoService = infoService)` should be removed as well. However, I fail to see the benefit as it violates the code legibility a tiny bit.